src/Reflection/ExtendedMethodReflection.php

```diff
@@ -14,6 +14,8 @@
  * and its methods in their code.
  *
  * Methods on ExtendedMethodReflection are subject to change.
+ *
+ * @api
```

Comment:
above this interface is this big comment, mentioning that this interface is not meant for re-use. I guess it should be dropped now?

Reply:
I'll rewrite it after merge.